src/topology/algebra/module/strong_topology.lean

```diff
@@ -249,49 +295,29 @@ variables {𝕜 : Type*} {𝕜₂ : Type*} {𝕜₃ : Type*} {𝕜₄ : Type*}
     [ring_hom_inv_pair σ₄₃ σ₃₄]
   [ring_hom_comp_triple σ₂₁ σ₁₄ σ₂₄] [ring_hom_comp_triple σ₂₄ σ₄₃ σ₂₃]
     [ring_hom_comp_triple σ₁₂ σ₂₃ σ₁₃] [ring_hom_comp_triple σ₁₃ σ₃₄ σ₁₄]
+    [ring_hom_comp_triple σ₂₃ σ₃₄ σ₂₄] [ring_hom_comp_triple σ₁₂ σ₂₄ σ₁₄]
+  [ring_hom_isometric σ₁₂] [ring_hom_isometric σ₂₁]
 
 include σ₁₄ σ₂₄ σ₁₃ σ₃₄ σ₂₁ σ₂₃
 
 /-- A pair of continuous (semi)linear equivalences generates a (semi)linear equivalence between the
 spaces of continuous (semi)linear maps. -/
-@[simps] def arrow_congrₛₗ (e₁₂ : E ≃SL[σ₁₂] F) (e₄₃ : H ≃SL[σ₄₃] G) :
```

Comment:
Are you sure we want to drop `arrow_congrₛₗ`? Why not both?

Reply:
Well the extra hypothesis are not actually adding any constraint IIRC (in the sense that they follow from the previous assumptions, but the typeclass system can't know that), so just using `arrow_congrSL` looks fine to me.